main.nf

```diff
@@ -1,352 +1,259 @@
 #!/usr/bin/env nextflow
```

Comment:
We are tending at the moment to provide the actual workflows in a separate folder called `workflows` and just have the `main.nf` call that workflow. See for example https://github.com/nf-core/viralrecon/tree/dev, or the RNAseq pipeline (https://github.com/nf-core/rnaseq), where `main.nf` basically only calls the `rnaseq.nf` workflow. This allows separation of the boilerplate code in `main.nf` from the actual pipeline code and will make template updates merges in the `main.nf` much easier.
However as you just have one workflow for now, you can also leave it in the main.nf if you prefer for now.